cmd/csi-snapshotter/main.go

```diff
@@ -229,6 +230,15 @@ func main() {
 	snapShotter := snapshotter.NewSnapshotter(csiConn)
 	var groupSnapshotter group_snapshotter.GroupSnapshotter
 	if *enableVolumeGroupSnapshots {
+		supportsCreateVolumeGroupSnapshot, err := supportsGroupControllerCreateVolumeGroupSnapshot(ctx, csiConn)
+		if err != nil {
+			klog.Errorf("error determining if driver supports create/delete group snapshot operations: %v", err)
+			os.Exit(1)
+		}
+		if !supportsCreateVolumeGroupSnapshot {
+			klog.Errorf("CSI driver %s does not support GroupControllerCreateVolumeGroupSnapshot", driverName)
+			os.Exit(1)
```

Comment:
This should not exit.  It is very likely that a CSI driver may combine the controller plugin and group controller plugin as one.

Reply:
@xing-yang 

If this parameter `enable-volume-group-snapshots` is specified as true, but during the verification process, it is found that the group snapshot is not supported by the csi-plugins , then how should we deal with it? Print a warning log or exit directly?

Reply:
Print a warning message but don't exit.